theme/components/sections/home/banner.html.twig

```diff
@@ -13,13 +14,61 @@
 {% endembed %}
 
 <div class="home-page__marquee-wrapper">
-    {{ include('partials/ui/marquee', { content: 'components.sections.home.banner.marquee'|localize }) }}
+    {% set content = marquee ? marquee : 'components.sections.home.banner.marquee'|localize %}
```

Comment:
The `marquee` text will fallback to the default text that is localized, if the editor controls modify this value it'll skip localizing. 

there's ongoing discussion about this, but in the current state the `localize` filter will not work with editor modified values because `localize` expects a localization key but the editor writes raw values in which makes the filter omit the value cc @kevinlee11